Kani throws a python exception if there is an error in the program

<!--
Please report security concerns to us on the
[AWS vulnerability reporting](http://aws.amazon.com/security/vulnerability-reporting/) page.
-->

<!--
Thank you for filing a bug report! 🐛
Please provide a short summary of the issue, along with the information necessary to replicate.
-->

I tried this code:
<!--
If the reproducing code is small, please post it here.
Otherwise, please attach the relevant files to this issue.
Alternatively, post a link to the repository and branch that exposes the issue.
-->
```rust
fn main() {
    assert!(x);
}
```

using the following command line invocation:

```
kani triv.rs
```

with Kani version: `6dc03c2eb53`

I expected to see this happen:
```
error[E0425]: cannot find value `x` in this scope
 --> triv.rs:2:13
  |
2 |     assert!(x);
  |             ^ not found in this scope

```

<!--
If Kani crashed, please include a backtrace in the code block by setting
`RUST_BACKTRACE=1` in your environment. E.g. `RUST_BACKTRACE=1 kani ...`
-->
Instead, this happened:
```
$ kani triv.rs 
Traceback (most recent call last):
  File "/home/ubuntu/git/kani/scripts/kani.py", line 197, in run_cmd
    returncode = cbmc_json_parser.transform_cbmc_output(stdout, output_style)
  File "/home/ubuntu/git/kani/scripts/cbmc_json_parser.py", line 121, in transform_cbmc_output
    raise Exception("CBMC Crashed - Unable to present Result")
Exception: CBMC Crashed - Unable to present Result

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/git/kani/scripts/kani", line 201, in <module>
    sys.exit(main())
  File "/home/ubuntu/git/kani/scripts/kani", line 59, in main
    if EXIT_CODE_SUCCESS != kani.compile_single_rust_file(args.input, base, symbol_table_json_filename, args):
  File "/home/ubuntu/git/kani/scripts/kani.py", line 261, in compile_single_rust_file
    return run_cmd(
  File "/home/ubuntu/git/kani/scripts/kani.py", line 199, in run_cmd
    raise Exception("JSON Parsing Error")
Exception: JSON Parsing Error
```
The CBMC json parser script should not be invoked if there was an error in compilation.

--- a/scripts/kani.py
+++ b/scripts/kani.py
@@ -192,7 +192,7 @@ def run_cmd(
     # Write to stdout if specified, or if failure, or verbose or debug
     if (output_to == "stdout" or process.returncode != EXIT_CODE_SUCCESS or verbose or debug) and not quiet:
         # By Default, the flag passed is the old output style
-        if (output_style != kani_flags.OutputStyle.OLD and not dry_run):
+        if (label == "cbmc" and output_style != kani_flags.OutputStyle.OLD and not dry_run):
             try:
                 returncode = cbmc_json_parser.transform_cbmc_output(stdout, output_style)
             except BaseException:

